llvm/include/llvm/Passes/MachinePassRegistry.def

```diff
@@ -151,6 +152,7 @@ MACHINE_FUNCTION_PASS("print<machine-branch-prob>",
 MACHINE_FUNCTION_PASS("print<machine-dom-tree>",
                       MachineDominatorTreePrinterPass(dbgs()))
 MACHINE_FUNCTION_PASS("print<machine-loops>", MachineLoopPrinterPass(dbgs()))
+MACHINE_FUNCTION_PASS("print<machine-cycles>", MachineCycleInfoPrinterPass(dbgs()))
```

Comment:
Alphabetical order.

Reply:
The legacy analysis is already named `MachineCycleInfoWrapperPass`

The printer also runs the analysis to print it.